src/python_testing/TC_OpstateCommon.py

```diff
@@ -0,0 +1,1191 @@
+                                                          expected_value=cluster.Enums.ErrorStateEnum.kCommandInvalidInState)
+            else:
+                self.skip_step("7h")
+
+    ############################
+    #   TEST CASE 2.2
+    ############################
+    def STEPS_TC_OPSTATE_BASE_2_2(self) -> list[TestStep]:
+        steps = [TestStep(1, "Commissioning, already done", is_commissioning=True),
+                 TestStep(2, "Manually put the DUT into a state wherein it can receive a Start Command"),
+                 TestStep(3, "TH reads from the DUT the OperationalStateList attribute"),
+                 TestStep(4, "TH sends Start command to the DUT"),
+                 TestStep(5, "TH reads from the DUT the OperationalState attribute"),
+                 TestStep(6, "TH reads from the DUT the OperationalError attribute"),
+                 TestStep(7, "TH reads from the DUT the CountdownTime attribute"),
+                 TestStep(8, "TH reads from the DUT the PhaseList attribute"),
+                 TestStep(9, "TH reads from the DUT the CurrentPhase attribute"),
+                 TestStep(10, "TH waits for {PIXIT.WAITTIME.COUNTDOWN}"),
+                 TestStep(11, "TH reads from the DUT the CountdownTime attribute"),
+                 TestStep(12, "TH sends Start command to the DUT"),
+                 TestStep(13, "TH sends Stop command to the DUT"),
+                 TestStep(14, "TH reads from the DUT the OperationalState attribute"),
+                 TestStep(15, "TH sends Stop command to the DUT"),
+                 TestStep(16, "Manually put the DUT into a state wherein it cannot receive a Start Command"),
+                 TestStep(17, "TH sends Start command to the DUT")
+                 ]
+        return steps
+
+    async def TEST_TC_OPSTATE_BASE_2_2(self, endpoint=1):
+        cluster = self.test_info.cluster
+        attributes = cluster.Attributes
+        commands = cluster.Commands
+
+        self.init_test()
+
+        asserts.assert_true('PIXIT.WAITTIME.COUNTDOWN' in self.matter_test_config.global_test_params,
+                            "PIXIT.WAITTIME.COUNTDOWN must be included on the command line in "
+                            "the --int-arg flag as PIXIT.WAITTIME.COUNTDOWN:<wait time>")
+
+        wait_time = self.matter_test_config.global_test_params['PIXIT.WAITTIME.COUNTDOWN']
+
+        if wait_time == 0:
+            asserts.fail("PIXIT.WAITTIME.COUNTDOWN shall be higher than 0.")
+
+        # STEP 1: Commission DUT to TH (can be skipped if done in a preceding test)
+        self.step(1)
+
+        # STEP 2: Manually put the DUT into a state wherein it can receive a Start Command
+        self.step(2)
+        self.send_manual_or_pipe_command('{"Name":"OperationalStateChange", "Device":"%s", "Operation":"OnFault", "Param": %s}' %
+                                         (self.device, cluster.Enums.ErrorStateEnum.kNoError))
+        if self.is_ci:
+            time.sleep(1)
```

Comment:
- This should be part of `send_manual_or_pipe_command`, it would save a lot of repetition.
- Is 1s too long? It likely, if done over local machine for CI. Suggest 0.1. Would have been better even to have acknowledged protocol...

Reply:
done. I kept `sleep` for now, until further acknowledgment protocol implementation.